### PR TITLE
3.x - Remove deprecated shell tasks

### PR DESCRIPTION
**Pull request by othercorey**
Removing deprecated classes to avoid dependencies before feature work begins.

### CODE REVIEW

File: src/Command/EntryCommand.php [hunk 1]
```diff
@@ -1,231 +0,0 @@
...
-use Cake\Console\Exception\ConsoleException;
-use Cake\Console\Shell;
-use Cake\Core\Configure;
-use Cake\Core\Plugin as CorePlugin;
-use Cake\Utility\Inflector;
-
-/**
- * Command that provides help and an entry point to bake tools.
- */
-class EntryCommand extends Command implements CommandCollectionAwareInterface
```
Comment (ADmad) [inline src/Command/EntryCommand.php:36]: What will be the output for `cake bake` if this command is removed?

Comment (othercorey) [reply src/Command/EntryCommand.php:36]: We probably need a help command. I didn't notice it performed both functions.

Comment (ADmad) [reply src/Command/EntryCommand.php:36]: Well the command's name doesn't really matter. We can just remove the unneeded parts of this command and avoid unnecessary code churn.

Comment (othercorey) [reply src/Command/EntryCommand.php:36]: Agreed.

**Comment by othercorey** [comment]
Realized that unless Bake 3 is CakePHP 5+, we can't prevent the shell tasks from showing up in the command list.

**Pull request closed without merging**